pyenvbuilder/commands/check.py

```
@@ -46,16 +61,28 @@ def yaml_loader(file_path):
             logging.error('Loading YAML file error:\n {}'.format(err))
 
 
-def yaml_validator(yaml_file):
+def yaml_validator(yml_file):
     '''
     Validates and YAML file against a schema
+    and returns True if valid, False otherwise
```

Can we also return the error string along with the status?